### ISSUE TITLE
cURL integration?

### ISSUE THREAD

**Issue by beriberikix** [obgm/libcoap#593]
https://curl.se/mail/lib-2018-05/0054.html

A few years ago Olaf and others brought up adding libcoap to libcurl/cURL. At the time there seemed lukewarm interest. More recently, [MQTT](https://daniel.haxx.se/blog/2020/04/14/curl-mqtt-true/) was added and it may be a good opportunity to revisit the proposal. Is there any interest in the community?

**Comment by obgm** [comment]
Yes, I still have it on my TODO-list (especially since libcoap has improved a lot since). Currently, lack of time has been the limiting factor.

**Comment by beriberikix** [comment]
Very understandable :) and I agree, there has been a lot of improvements to warrant a new discussion!

**Comment by beriberikix** [comment]
Happy new year! I thought I'd check in and see if there's renewed interest in trying to get this into curl.

**Comment by obgm** [comment]
Hi there, yes (more than ever). I will work on this and come up with a proof-of-concept implementation shortly. I had this outsourced a few months ago but the volunteer seems to have disappeared.
